cmd/kes/identity.go

```diff
@@ -249,9 +249,9 @@ func newIdentityCmd(args []string) {
 		if expiry > 0 {
 			cli.Fatalf("'--expiry' requires a private key and certificate file. Set the '--cert' and '--key' flag")
 		}
-		if len(IPs) > 0 {
-			cli.Fatalf("'--ip' requires a private key and certificate file. Set the '--cert' and '--key' flag")
-		}
+		// if len(IPs) > 0 {
```

Comment:
do we need commented lines?

Reply:
Thnx for pointing that out. I had left some debug code in there. Forgot to check my own PR 👎🏻